diff --git a/xbmc/network/upnp/UPnPServer.cpp b/xbmc/network/upnp/UPnPServer.cpp
index de879ff..92feb82 100644
--- a/xbmc/network/upnp/UPnPServer.cpp
+++ b/xbmc/network/upnp/UPnPServer.cpp
@@ -983,6 +983,11 @@ static NPT_String TranslateWMPObjectId(NPT_String id)
         return NPT_SUCCESS;
     }
 
+    if (file_path.Left(13).Compare("image://video", true) == 0 || file_path.Left(13).Compare("image://music", true) == 0) {
+        response.SetStatus(404, "File Not Found");
+        return NPT_SUCCESS;
+    }
+
     if(URIUtils::IsURL((const char*)file_path))
     {
       CStdString disp = "inline; filename=\"" + URIUtils::GetFileName((const char*)file_path) + "\"";
